web/themes/custom/recipe_theme/package-lock.json

```diff
@@ -0,0 +1,11389 @@
+{
```

Comment:
Let's remove this file. As we are using `yarn` for package management, this lock file is not needed.

Reply:
Okay, done